src/WinUI/ArcGISRuntime.WinUI.Viewer/Samples/Data/ToggleBetweenFeatureRequestModes/readme.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "category": "Data",
+    "description": "Use different feature request modes to populate the map from a service feature table.",
+    "formal_name": "ToggleBetweenFeatureRequestModes",
+    "ignore": false,
+    "images": [
+        "ToggleBetweenFeatureRequestModes.jpg"
+    ],
+    "keywords": ["cache",
```

Comment:
This metadata is out of sync. If you run the updated version of the script from the latest v.next it will resolve this.